packages/layout/components/App/App.module.css

```diff
@@ -0,0 +1,13 @@
+@layer custom {
```

Comment:
I'm convinced that we need to add an extra layer for "default" Seven theming... unsure about the name, as usual.

Reply:
Let's do it in a subsequent PR, I don't want to hold this more.